src/bench/checkqueue.cpp

```diff
@@ -26,6 +26,10 @@ static const unsigned int QUEUE_BATCH_SIZE = 128;
 // and there is a little bit of work done between calls to Add.
 static void CCheckQueueSpeedPrevectorJob(benchmark::Bench& bench)
 {
+    if (GetNumCores() < MIN_CORES) {
```

Comment:
4f9164389a078f64258f3d7f2ecda5ab68968e8c

nit, single line and also add comment.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/19710#issuecomment-680261215).